.idea/compiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file from the PR? Can you also add the .idea/compiler.xml to the .gitignore file?

Reply:
I have removed the file from PR and have also added to .gitignore file. Pls review the PR @katielevy1